libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -79,11 +79,12 @@
     #endif
 
     #define DECLARE_FOR_ALL_TYPES(foo)\
-	   foo(char) json_nothrow;	 foo(unsigned char) json_nothrow;\
-	   foo(short) json_nothrow;	 foo(unsigned short) json_nothrow;\
-	   foo(int) json_nothrow;	 foo(unsigned int) json_nothrow;\
-	   foo(long) json_nothrow;	 foo(unsigned long) json_nothrow;\
-	   foo(float) json_nothrow;	 foo(double) json_nothrow;\
+       foo(char) json_nothrow;      foo(unsigned char) json_nothrow;\
```

Comment:
In facts, libjson formatting is an evil mix of spaces and tabs. Whenever we need to modifiy the source (which, I hope, is never going to happen again), I think we should just reproduce whatever they did and be happy with that.